pythonsdk/src/main/python/vantiqservicesdk.py

```diff
@@ -182,12 +183,22 @@ async def __process_message(self, websocket: WebSocket, msg_bytes: bytes) -> Non
             request_timer = self.__get_resource_metric(self._resources_executions, procedure_name)
             with request_timer.time():
                 result = await self.__invoke(procedure_name, params)
-                response["result"] = result
-
-            # Serialize to JSON and encode
-            text = json.dumps(response, separators=(",", ":"), ensure_ascii=False,
-                              default=lambda obj: self.to_json(obj))
-            encoded_response = text.encode("utf-8")
+                if isinstance(result, AsyncIterator):
```

Comment:
Protocol already supports sending incremental results and saving the "isEOF" for the last one.